content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -160,7 +160,7 @@ The general workflow of a device plugin includes the following steps:
    The processing of the fully-qualified CDI device names by the Device Manager requires
    that the `DevicePluginCDIDevices` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
    is enabled for both the kubelet and the kube-apiserver. This was added as an alpha feature in Kubernetes
-   v1.28 and graduated to beta in v1.29.
+   v1.28, graduated to beta in v1.29 and to GA in v1.30.
```

Comment:
```suggestion
   v1.28, graduated to beta in v1.29 and to GA in v1.31.
```

Reply:
You will also need to update the file under `content/en/docs/reference/command-line-tools-reference/feature-gates/`.

Reply:
done, thanks